### PR TITLE
feat(managed-delivery): add unresolvable status to glossary

### PR DESCRIPTION
**Pull request by erikmunson**
Adding the documentation companion to https://github.com/spinnaker/deck/pull/8189. Also did a very small wording tweak across the statuses.

### CODE REVIEW

File: guides/user/managed-delivery/resource-status/index.md [hunk 1]
```diff
@@ -128,18 +128,32 @@ When you resume management the status will change to [Resumed](/guides/user/mana
...
+  figure
+  image_path="./resource-status-currently-unresolvable.png"
+%}
+
+When a resource has this status, it means something Spinnaker requires to properly manage the resource is either not ready yet or temporarily experiencing disruption. Automatic action won't be taken while in this status, but it doesn't necessarily mean something is broken. Some examples of things that might lead to a resource entering this status:
+
+  - A downstream system (like a container registry or cloud provider) temporarily rate limiting or experiencing availability issues
+  - A required part of the resource's desired state (like a VM image) not being available yet
+
+If you see this status and want to investigate the cause, you can click "History" to view additional details and understand what else was happening when the issue began. While you may decide to take some action as a result of this status, it's unlikely the issue requires manual intervention. Once the issue is resolved the resource will transition to whatever status is applicable at the time based on its desired state.
```
Comment (gal-yardeni) [inline guides/user/managed-delivery/resource-status/index.md:147]: can we add a screenshot/link when we send the user to check the `history`?

File: guides/user/managed-delivery/resource-status/index.md [hunk 2]
```diff
@@ -128,18 +128,32 @@ When you resume management the status will change to [Resumed](/guides/user/mana
...
+### Currently unresolvable
+{%
+  include
+  figure
+  image_path="./resource-status-currently-unresolvable.png"
+%}
+
+When a resource has this status, it means something Spinnaker requires to properly manage the resource is either not ready yet or temporarily experiencing disruption. Automatic action won't be taken while in this status, but it doesn't necessarily mean something is broken. Some examples of things that might lead to a resource entering this status:
+
+  - A downstream system (like a container registry or cloud provider) temporarily rate limiting or experiencing availability issues
```
Comment (gal-yardeni) [inline guides/user/managed-delivery/resource-status/index.md:144]: Love the examples here, I think it's super helpful! I wonder if we can add some resources here that can help users to figure out what is wrong, like troubleshooting links? (I don't really know if we have something like this)

**Review by gal-yardeni — APPROVED** [review]
LGTM :)

File: guides/user/managed-delivery/resource-status/index.md (hunk 2 above)
Comment (erikmunson) [reply guides/user/managed-delivery/resource-status/index.md:144]: yeah i share that desire — sadly it's pretty specific to the error and many times will involve company-internal systems. I think we _can_ make an impact on this though by adding helpful guidance and recommendations to the error messages we add to history events where useful!

File: guides/user/managed-delivery/resource-status/index.md (hunk 1 above)
Comment (erikmunson) [reply guides/user/managed-delivery/resource-status/index.md:147]: you mean like a picture of what the history event looks like when a resource is in this status? If so, I have plans for that! I'm thinking about having a separate doc page about how to use the history view with a guide similar to this one covering each history event type — once that's added we can link between this page and that page so folks can understand how the two connect to each other.

Comment (gal-yardeni) [reply guides/user/managed-delivery/resource-status/index.md:147]: Sounds AMAZING! can't wait ❤️

**Pull request merged**